modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -341,14 +365,100 @@ dbms.security.oidc.google.token_params=client_secret=GOCSPX-v4cGkygPJvm3Sjjbc0hv
 dbms.security.oidc.google.config=token_type_principal=id_token;token_type_authentication=id_token
 ----
 
-. Log in with your Google SSO credentials using the email address and get the `admin` role when doing so:
-+
-image::sso-configuration-tutorials/oidc-azure-successful-login.png[title="Entra OIDC successful login"]
-+
-[NOTE]
-====
-The native authentication is disabled to prevent someone from logging in to _alice@neo4j-test.com_ with the set password.
-====
+There are then two ways to create the user in the database (which is required to give the users roles from native authorization):
+
+. Using Auth Providers
+xref:authentication-authorization/auth-providers.adoc[Auth Providers] allow us to specify auth providers at the user-level which is useful in this scenario. This approach allows us to have an `admin` user who can authenticate natively, and can then create less privileged users who may only authenticate using `oidc-google` and will receive the roles granted to them using `native` authorization.
+
+.. Change the configuration to allow `native` authentication (for use only by the `admin` user):
+
+[source, properties]
+----
+dbms.security.authentication_providers=oidc-google, native
+----
+
+.. Set the `dbms.security.require_local_user` configuration setting to `true` in the _neo4j.conf_ file. This will switch to User Auth Providers mode whereby users can only authenticate and authorize if they have a corresponding Auth Provider in the database.
+
+[source, properties]
+----
+dbms.security.require_local_user=true
+----
+
+
+.. Create a user who can authenticate and authorize using `oidc-google` (they will not be able to authenticate using `native` (username and password) authentication, only `oidc-google`):
+
+[source, cypher, role=noplay]
+----
+CREATE USER jake
+SET HOME DATABASE 'jakesHomeDb'
+SET AUTH 'oidc-google' {SET ID 'jakesUniqueGoogleUserId'} // this must match the claim that you configured via dbms.security.oidc.google.claims.username
+----
+
+.. Grant the user roles using native authorization.
+
+[source, cypher, role=noplay]
+----
+GRANT ROLE reader TO jake
+----
+
+The user will implicitly receive `native` authorization because `native` is in the list of authorization providers and you have explicitly granted the user a `ROLE`.
+
+You can now disable native authentication for the database completely:
```

Comment:
Having a database with a single user with the role reader doesn't seem so helpful (or it also has an admin user that might no longer be able to log in) XD

I guess it's more along the lines of 'once everything is setup to your liking' you can disable the native authentication, but it reads a bit odd :P (especially since there is no mention of updating the admin user to be allowed to log in externally with it's native authorization so it could continue its work. Though it might be in any of the unchanged parts of the doc, I have not read them :P)

Reply:
Thanks, yes. I rephrased this and added a note (currently line 380) to clarify the existence of the admin user.